packages/firestore/test/unit/local/local_store.test.ts

```diff
@@ -1654,6 +1688,40 @@ function genericLocalStoreTests(
       .finish();
   });
 
+  it('handles saving and loading named queries', async () => {
+    return expectLocalStore()
+      .after(namedQuery('test', query('coll'), 'FIRST', SnapshotVersion.min()))
```

Comment:
It is not clear what 'test' and 'FIRST' are. Can you make it clear (possible by using the enum constants)?